framework/generated/dx12_generators/dx12_base_generator.py

```diff
@@ -59,7 +59,7 @@ class Dx12BaseGenerator(BaseGenerator):
         [
             'ID3D12GraphicsCommandList_OMSetRenderTargets',
             'pRenderTargetDescriptors',
-            'RTsSingleHandleToDescriptorRange ? 1 : NumRenderTargetDescriptors'
+            '(NumRenderTargetDescriptors ? (RTsSingleHandleToDescriptorRange ? 1 : NumRenderTargetDescriptors) : 0)'
```

Comment:
This line is inherited from the platform-independent PR https://github.com/LunarG/gfxreconstruct/pull/939, which should be merged before this PR (should they pass review).